package.json

```diff
@@ -23,7 +23,6 @@
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
   "bin": {
-    "tt-aggregate-translations": "bin/aggregate-translations.js",
```

Comment:
Related to the comment on the bin file, wondering if we can keep this for passivity and route the file required from the bin file in terra-toolkit to the terra-aggregate-translations file.

Reply:
Based on https://github.com/cerner/terra-toolkit/pull/275#discussion_r279482266, you can disregard this comment.